### PR TITLE
Translated 'links and images'

### PR DESCRIPTION
**Pull request by gorkemdolcek**
html-css/html-foundations/lins and images.md dökümanının türkçeye çevirilmesi.

### CODE REVIEW

File: temel_bilgiler/html_css/html-temelleri/linkler ve resimler.md [hunk 1]
```diff
@@ -0,0 +1,268 @@
...
+
+Açılış ve kapanış etiketiyle içeriği sarmak yerine src özelliğini kullanarak bir görüntüyü sayfaya yerleştirir. src özelliği tarayıcıya görüntü dosyasının nerede bulunduğunu söyler. src özelliği bağlantı öğeleri için href özelliği gibi çalışır. Hem mutlak hem de göreceli yol kullanarak bir görüntü yerleştirebilir.
+
+Örneğin The Odin Project sitesinde bulunan bir görüntüyü mutlak yol kullanarak görüntüleyebiliriz:
+
+<p class="codepen" data-height="300" data-theme-id="dark" data-default-tab="html,result" data-slug-hash="gORbExZ" data-user="TheOdinProjectExamples" style="height: 300px; box-sizing: border-box; display: flex; align-items: center; justify-content: center; border: 2px solid; margin: 1em 0; padding: 1em;">
+  <span>See the Pen <a href="https://codepen.io/TheOdinProjectExamples/pen/gORbExZ">
+  absolute-path-image</a> by TheOdinProject (<a href="https://codepen.io/TheOdinProjectExamples">@TheOdinProjectExamples</a>)
+  on <a href="https://codepen.io">CodePen</a>.</span>
```
Comment (cansirin) [inline temel_bilgiler/html_css/html-temelleri/linkler ve resimler.md]:
```suggestion
<span><a href="https://codepen.io">CodePen</a>'de TheOdinProject 
(<a href="https://codepen.io/TheOdinProjectExamples">@TheOdinProjectExamples</a>) tarafından hazırlanan 
<a href="https://codepen.io/TheOdinProjectExamples/pen/gORbExZ">absolute-path-image örneğine</a> göz atın.</span>
```

**Comment by LiberaTeMetuMortis** [comment]
I think we have to replace "nitelik" and "öznitelik" words with "özellik" word. Since we translated "attribute" word as "özellik" in https://github.com/kamp-us/turkce-odin-project/blob/main/temel_bilgiler/html_css/html-temelleri/html-boilerplate.md and it is more natural.

**Comment by gorkemdolcek** [comment]
> I think we have to replace "nitelik" and "öznitelik" words with "özellik" word. Since we translated "attribute" word as "özellik" in https://github.com/kamp-us/turkce-odin-project/blob/main/temel_bilgiler/html_css/html-temelleri/html-boilerplate.md and it is more natural.

You are right, I didn't see such a thing in the translation reference list. If @cansirin agrees, we can fix it.

**Comment by cansirin** [comment]
> > I think we have to replace "nitelik" and "öznitelik" words with "özellik" word. Since we translated "attribute" word as "özellik" in https://github.com/kamp-us/turkce-odin-project/blob/main/temel_bilgiler/html_css/html-temelleri/html-boilerplate.md and it is more natural.
> 
> You are right, I didn't see such a thing in the translation reference list. If @cansirin agrees, we can fix it.

Yep let's do this :) Thanks for the support and contribution!

**Review by cansirin — APPROVED** [review]
Perfecto 🚀 💯

**Pull request merged**